reana_cluster/backends/kubernetes/templates/backend_conf.yaml

```diff
@@ -3,11 +3,13 @@
 {% include 'deployments/workflow-controller-template.yaml' %}
 {% include 'deployments/db-template.yaml' %}
 {% include 'deployments/cache-template.yaml' %}
+{% include 'deployments/ui-template.yaml' %}
```

Comment:
Make REANA UI optional (default false, as it is currently only useful on CERN deployment).